src/neural-network.js

```diff
@@ -60,6 +75,16 @@ export default class NeuralNetwork {
     }
   }
 
+  /**
+   * 
+   * validateTrainingOptions is set true as default
+   *      When true: setting a training option outside expected range will throw an error
+   *      When false: setting a training option outside expected range will log out and continue
+   */
+  get validateTrainingOptions() { return this._isStrict; }
+  set validateTrainingOptions(s) { this._isStrict = s; }
```

Comment:
pass in an object?

Reply:
yeah it was a setter/getter but I got rid of that (since there aren't any intended side effects.